subgraphs/compound-forks/protocols/venus/src/mapping.ts

```diff
@@ -293,6 +342,103 @@ export function handleTransfer(event: Transfer): void {
   );
 }
 
+// update market reward amounts based on event data.
+export function handleVenusSpeedUpdated(event: VenusSpeedUpdated): void {
+  const marketAddress = event.params.vToken;
+  const speed = event.params.newSpeed;
+  const rewards = getRewardsPerDay(
+    event.block.timestamp,
+    event.block.number,
+    speed.toBigDecimal(),
+    RewardIntervalType.BLOCK
+  ).times(BigDecimal.fromString(DAYS_PER_YEAR.toString()));
```

Comment:
Rewards should be recorded on a per day basis, that is likely why they are so high on subgraphs.xyz

Reply:
Damn, not sure why I put it to a year. I was convinced we normalized them to a year 😅